### ISSUE TITLE
After checking out a PR and modifying the files locally, support converting the changes to GitHub PR suggestion comments

### ISSUE THREAD

**Issue by tiangolo** [microsoft/vscode-pull-request-github#6034]
## Description

After checking out a PR and modifying the files locally, support converting the changes to GitHub PR suggestion comments.

## Current Workflow

In my workflows reviewing PRs, I normally use the VS Code GitHub Pull Request option to check out a PR by number.

Then I see the changes in files, and I start adjusting them in the editor. In many cases running and debugging the code locally until it's ready.

Once it is done, I end up with changed files in the "Source Control" tab (files not yet staged in git), but I don't have an easy way to convert those changes into GitHub PR inline suggestion comments.

The options I have are:

### Copy Paste in VS Code

* Go through each change I made to the files and manually copy-paste it
* Switch the tab from "Source Control" to "GitHub Pull Request"
* Manually find the same location of the change (difficult to remember sometimes)
* Add a comment, make it a suggestion, paste the contents copied
* Go back to the "GitHub Pull Request" tab, copy-paste the next section, and do the same process

### Copy Paste to GitHub

* Go through each change I made to the files and manually copy-paste it
* Open a browser tab with GitHub, find the same file in the PR, find the same section
* Add a comment, make it a suggestion, paste the contents copied
* Go back to VS Code, copy the next piece of code and start again

### Commit on Top

I can also just commit my changes on top of the PR and push them.

This is the simplest option, so, for several changes, that's what I end up doing. But in many cases, it would be better to make them as suggestions that I can show the original author, explain what needs to be changed, etc.

## What I Would Like

I would like to be able to make changes directly in the code, in VS Code, and then have an integrated way to convert those local changes to suggestion comments.

### Possible Worflows

* Maybe each of my local changes could be shown in some way in the "GitHub Pull Request" with a button to "add as suggestion".

* Or maybe a button to "add a suggestion comment to PR" could be shown in the "Source Control" tab.

* Or maybe I could git stage the changes I would like to suggest and there could be a button or command to "send all staged changes as inline code suggestions to PR" that starts a PR review with all my comments, and then I can go to GitHub on the browser and update them with more comments or edit them further if necessary.

**Comment by alexr00** [comment]
Duplicate of https://github.com/microsoft/vscode-pull-request-github/issues/5256
